package.json

```diff
@@ -72,6 +72,7 @@
     "puppeteer": "latest",
     "standard-version": "latest",
     "ts-jest": "latest",
+    "ts-loader": "latest",
```

Comment:
Why did you add this?

Reply:
I thought it was a missing dependency at first, but turns out I don't need it, so I've removed it